src/core/psxemulator.cc

```diff
@@ -94,7 +94,7 @@ void PCSX::Emulator::EmuReset() {
     m_cheats->FreeCheatSearchResults();
     m_cheats->FreeCheatSearchMem();
     m_psxMem->psxMemReset();
-
+    m_spu->resetCaptureBuffer();
```

Comment:
This may only be valid for a hard reset, but I don't think it matters that much.

Reply:
Can also put this for soft reset too.

Reply:
Either way is fine, really. This buffer ought to be reset pretty quickly, and I don't think we're properly resetting the SPU ram on reset at the moment anyway. Still a lot of cleanup to do :D